### PR TITLE
fix: worklet and node rust

### PR DESCRIPTION
**Pull request by hylcore-V**

### CODE REVIEW

File: macros/src/node.rs [hunk 1]
```diff
@@ -2,42 +2,56 @@ use proc_macro2::{Ident as ProcIdent, Span};
...
-            initial_state: Option<<#ident as waw::worklet::AudioModule>::InitialState>
-          ) -> Result<#node_ident, wasm_bindgen::JsValue> {
-              let result = waw::node::Node::<#ident>::create(ctx, initial_state).await?;
-              Ok(#node_ident(result))
-          }
-          pub fn node(&self) -> Result<waw::web_sys::AudioWorkletNode, wasm_bindgen::JsValue> {
-            Ok(self.0.inner.clone())
-          }
-
-          pub fn get_param(&self, param: <#ident as waw::worklet::AudioModule>::Param) -> waw::web_sys::AudioParam {
```
Comment (lunarforky-z) [inline macros/src/node.rs:26]: in this code part, i think it is good if u add the comment for easy understanding

File: macros/src/node.rs [hunk 2]
```diff
@@ -2,42 +2,56 @@ use proc_macro2::{Ident as ProcIdent, Span};
...
-            self.0.get_param(param)
-          }
-
-          pub fn command(&self, message: <#ident as waw::worklet::AudioModule>::Command) {
-              self.0.command(message)
-          }
-
-          pub fn subscribe(&mut self, callback: waw::utils::callback::Callback<<#ident as waw::worklet::AudioModule>::Event>) {
-              self.0.subscribe(callback.0)
```
Comment (lunarforky-z) [inline macros/src/node.rs:35]: this part too

**Review by lunarforky-z — CHANGES_REQUESTED** [review]

File: macros/src/node.rs (hunk 2 above)
Comment (hylcore-V) [reply macros/src/node.rs:35]: i have just push the modify code for ur feedback

**Review by lunarforky-z — APPROVED** [review]
great!
you comment looks good and can make customers easy understand of the feature of the code

**Comment by hylcore-V** [comment]
thank you
could u please merge my PR?

**Comment by lunarforky-z** [comment]
> thank you could u please merge my PR?

okay
i will merged it.
thank you for your work

**Pull request merged**